sdk/webpubsub/Azure.Messaging.WebPubSub.Client/README.md

```diff
@@ -36,13 +36,23 @@ dotnet add package Azure.Messaging.WebPubSub.Client
 
 ### Authenticate the client
 
-In order to interact with the service, you'll need to create an instance of the `WebPubSubClient` class. To make this possible, you'll need an access token. You can copy and paste an access token from Azure portal.
+Client uses a Client Access URL to connect and authenticate with the service. The Uri follow the patten as `wss://<service_name>.webpubsub.azure.com/client/hubs/<hub_name>?access_token=<token>`. The client has some different ways to get Client Access URL. As a quick start, you can copy and paste from Azure Portal, and for production, you usually need a negotiation server to generate the Uri.
+
+#### Use Client Access URL from Azure Portal
+
+As a quick start, you can go to the Portal and copy the **Client Access URL** from **Key** blade.
+
+![get_client_url](https://learn.microsoft.com/azure/azure-web-pubsub/media/howto-websocket-connect/generate-client-url.png)
+
+As shown in the diagram, the client will be granted the permission of sending message to the specific group and joining the specific group. Learn more about client permission, see [permissions](https://learn.microsoft.com/azure/azure-web-pubsub/reference-json-reliable-webpubsub-subprotocol#permissions)
 
 ```C# Snippet:WebPubSubClient_Construct
 var client = new WebPubSubClient(new Uri("<client-access-uri>"));
 ```
 
-And in production, you usually get `ClientAccessUri` from a negotiate server.
+#### Use negotiation server to generate Client Access URL
+
+In production, client usually fetch Client Access URL from a negotiation server. The server holds the connection string and generates Client Access URL through `WebPubSubServiceClient`. As an sample, the code snippet below just demostrate the how to generate the Client Access URL inside single process.
```

Comment:
Shall we add server sdk guide links to let user further discover details? I'm not sure if show `WebPubSubServiceClient` and `WebPubSubClient` together will make it confusing.
Or maybe add a table to introduce the difference clearly?